src/Model/Order/Order.php

```diff
@@ -160,6 +165,7 @@ public static function fromData(
         int $itemsCount,
         ?DateTimeInterface $promisedShippingTime,
         ?string $extraAttributes,
+        ?ExtraBillingAttributes $extraBillingAttributes,
```

Comment:
If new parameter is Nullable, should be better put it at the end. So you don't change the position of statuses that is not Nullable.